app/questionnaire/placeholder_transforms.py

```diff
@@ -27,7 +27,7 @@ def format_currency(self, number=None, currency='GBP'):
 
     def format_date(self, date_to_format, date_format):
         date_to_format = datetime.strptime(date_to_format, self.input_date_format)
-        return format_datetime(date_to_format, date_format, locale=self.locale)
+        return format_datetime(date_to_format, date_format, locale=self.language)
```

Comment:
Seems strange to use language here, when there is a locale variable in the class already. Shouldn't it be initialised to the correct value when the class is initialised instead?

Reply:
Additionally, we shouldn't be using locale on some methods and language on others, it should be the same throughout.

Reply:
I think I've made this more consistent with this commit.